nginx.conf

```diff
@@ -114,6 +114,13 @@
       proxy_read_timeout 600;
     }
 
+    # VictoriaMetrics
+
+    location /victoriametrics/ {
```

Comment:
Please remove trailing slash there and below. That will trigger a different behaviour in nginx. That's why we use it in Prometheus above and in Alertmanager below.

But please also test it :)

Reply:
`vmdb` started without path prefix, so this configuration will route request correctly ( root path at `vmdb` is not supported). Later we will add prometheus path prefix.

Reply:
Will configuration without trailing slashes route requests correctly? If yes, we should use it for consistency

Reply:
Nope, configuration without trailing slashes acts differently. 

 http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_pass